config/kubernetes/sig-pm/OWNERS

```diff
@@ -1,8 +0,0 @@
-# See the OWNERS docs at https://go.k8s.io/owners
-
-reviewers:
-  - sig-pm-leads
```

Comment:
Can you also remove this alias from OWNERS_ALIASES?

Reply:
@nikhita -- Removed! :)